ports/cs-libguarded/portfile.cmake

```
@@ -0,0 +1,30 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO copperspice/cs_libguarded
+    REF "libguarded-${VERSION}"
+    SHA512 fb54cce1be0ecbd363cdf7a8bdfcddd540737419aea702a29e5ee37143434f7431de1edb085bdfc2dc1eded9749dab40afd634582479118d2202c8d46b196c7d
+    HEAD_REF master
+)
+
+set(VCPKG_BUILD_TYPE release)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DBUILD_TESTS=OFF
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(
+    PACKAGE_NAME CsLibGuarded

```

There are discussions to enforce lower-case package names in `share`, and this option is used only for the directory name. Consider
```suggestion
    PACKAGE_NAME cslibguarded
```
And if the port would be named   `cslibguarded`, mathching the "CsLibGuarded" in the documentation, the option wouldn't be needed.
(I know repology/AUR/Conan have cs-libguarded. But packaging is still rare.)